### DISCUSSION TITLE
Concealing a specific child node

### DISCUSSION THREAD

**Discussion by Zynh0722** [nvim-treesitter/nvim-treesitter#4575] [Q&A]
I have a wonderful query, that captures precisely what I wish to capture, however I'm struggling to get the `#set!` directive to actually apply to the node I want it to apply to. 

```
(jsx_attribute 
  ((property_identifier) @name (#eq? @name "className")) 
    (string (string_fragment) @conceal (#set! conceal "󰇘")))
```

I just want the `string_fragment` captured by `@conceal` to be the thing concealed, currently no matter where I put that `#set!` or even if I add `@conceal` between `#set!` and `conceal` to mark the captureid, I either get no conceal at all, or the `property_identifier` gets concealed.

How do I get just the string_fragment to conceal? or even get it to conceal at all?

**Comment by rostero1** [comment]
I think this is the related issue: https://github.com/nvim-treesitter/nvim-treesitter/issues/4179
